.github/workflows/build-ale.yml

```diff
@@ -53,8 +53,11 @@ jobs:
       if: matrix.os != 'windows-latest'
       run: |
         mkdir build; cd build
-        cmake ../ -DUSE_SDL=ON
+        cmake ../ -DUSE_SDL=ON -DPython3_FIND_FRAMEWORK=NEVER
```

Comment:
New CMake policy seems to find Python 3.9 framework ahead of the installed 3.9 by Github actions, disable finding the Python framework.